transitions/loop.h

```diff
@@ -16,7 +16,35 @@ class TrLoop : public TR {
   bool done() { return false; }
 };
 
-// Usage: TrLoop<PULSE, TRANSITION, OUT>
+// Usage: TrLoopN<N, TRANSITION>
+// N: FUNCTION (number of Loops)
+// TRANSITION: TRANSITION
+// Return Value: TRANSITION
+// Runs the specified transition N times.
+
+template<class N, class TRANSITION>
+class TrLoopN : public TRANSITION {
```

Comment:
Actually, I guess this should be TrLoopNX, and we then add an alias called TrLoopN which takes an int.

Reply:
Added